code/Collector/PartialCacheCollector.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace LeKoala\DebugBar\Collector;
+
+use DebugBar\DataCollector\DataCollector;
+use DebugBar\DataCollector\Renderable;
+
+/**
+ * Collects data about the partial cache hits and misses during a SilverStripe request
+ */
+class PartialCacheCollector extends DataCollector implements Renderable
+{
+
+    /**
+     * Contains a list of all partial caches found.
+     * @var array
+     */
+    private static $templateCache = array();
```

Comment:
This might be better as protected. Private statics in SS are normally for config, whereas we generally prefer protected over private otherwise. Also - use `[]` to be consistent with other use in this class.